cli/command/context/show.go

```diff
@@ -25,10 +25,10 @@ func newShowCommand(dockerCli command.Cli) *cobra.Command {
 
 func runShow(dockerCli command.Cli) error {
 	context := dockerCli.CurrentContext()
-	metadata, err := dockerCli.ContextStore().GetMetadata(context)
+	fmt.Fprintln(dockerCli.Out(), context)
+	_, err := dockerCli.ContextStore().GetMetadata(context)
 	if err != nil {
 		return err
 	}
```

Comment:
From the discussion on https://github.com/docker/cli/pull/3868#discussion_r1025219461

> As this is a new command, we can still change; do we think we should produce an error _at all_ if the context wasn't found, or just "show the name of the context" (basically: this is what's _configured_ to use, but you'll learn if it works once you use it)?
> 
> We could still opt for printing the error (on STDERR) as a warning, but make the command always return a zero exit code ("success").